proposals/apply_polygon.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "apply_polygon",
+    "summary": "Apply a process to spatial chunks of a data cube",
+    "description": "The given data cube is chunked by the given polygons and applies the given process to each individual chunk.",
```

Comment:
This is a nitpick, but as a dask user the word "chunk" is overloaded for me. In dask, it would be possible that two of these polygons live within the same chunk. Also dask chunks are rectangular, which geometries are not necessarily. How about something like "The provided process is applied only to the subregions of the data cube described by the provided polygons."?

Reply:
Indeed, with the rename to `apply_polygon` we might want to steer away from using "chunk" all together

Reply:
Hmm... happy to optimize the wording, but "Apply a process to each given polygon in the datacube." sounds for me like you apply something on a vector data cube. I'll think about an alternative...

Reply:
Maybe it was also nitpicky from me. Given that we want to emphasize that the polygons are cut out of the input data cube first and then a process is applied to each polygon the longer sentence is probably better.